Add and update Repo Documentation

kozlowskik opened the pull request:
Updates were made to various READMEs within the repo to improve the content of the documents and make it easier to click on links.  
Also, added a new README in the `SystemTesting` directory to explain what is/will be going on in the directory.
***
This PR will close the following issues once the new features are pulled to Dev:
- Fixes #109 
- Fixes #110 
- Fixes #111

kozlowskik committed:
Removed the full http links from README

Removed the full http links from the README in root of the repo in the first paragraph. All of the links now are relative to the text that applies to that link.

kozlowskik committed:
Removed Unneeded Comment

Removed the original comment made about the link to Gradebook Requirements. Link has been fully functional. Comment has been removed.

kozlowskik committed:
Added Links to All Members Profiles

Modified the root README in the repo to make it so that all of our names in the members section are now links to all of our GitHub profiles.

kozlowskik committed:
Added System Testing and underlines

Added in the section on System testing on the file structure section of the README describing what the System Testing portion of the file structure will be like. Also added was some underlining of the different important directories that have information following them to make it so they stand out more.

kozlowskik committed:
Added badges to README

kozlowskik committed:
Updated ZenHub Badge links

Had the wrong links for the badge, it is still not showing correctly. Need to investigate

kozlowskik committed:
Merge branch 'master' into add-and-update-documentation

kozlowskik committed:
Create README for System Testing

Created the README for the System Testing Documentation directory to better exaplain what is/will be happening in the directory so that anyone who looks in the directory can understnad the purpose of the directory and subdirectories as well as the content that is within all directories in here.

kozlowskik committed:
Updated ZenHub Badge

Updated the ZenHub badge with the ZenHub logo from the ZenHub Press Kit as advised by a member of their customer service team so that we can have their logo in our README. A width of 75px was placed on the image to make it align with the rest of the badges. A comment was also added explaining all of this for anyone who looks at the code behind the README

kozlowskik committed:
Added Heading Comment And Updated List of Scripts

Added in 2 lines at the top of the document stating that GEEKS modified this document. and listed members names.
Updated the list of SQL scripts that need to be run to prepare the Database for running Gradebook.

kozlowskik committed:
Added Header Info and Updated Information

Added the same Header Comment as in the DB README.
Updated the Overview section to contain that the web server has been tested on Windows 10 and Windows Server 2019. Also updated the the section to include what internet browsers that the client works with, and added a comment stating that IE on Win 10 does not work

griffine commented on README.md:
```diff
@@ -28,10 +36,10 @@ The src directory will be split into two sub-directories:
 
 ## Members
-Bruno Desilva  
+[Bruno Desilva](https://github.com/DasilvaB)  
```
Bruno's last name is misspelled.

kozlowskik replied:
Corrected

griffine commented on src/db/README.md:
```diff
@@ -9,6 +9,9 @@ https://creativecommons.org/licenses/by-nc-sa/4.0/
 PROVIDED AS IS. NO WARRANTIES EXPRESSED OR IMPLIED. USE AT YOUR OWN RISK.
 
+Modified by **Team GEEKS** (*CS298-01 Spring 2019 @ WCSU*)  
+Bruno DaSilva, Elly Griffen, Chris Fitzgerald and Kenneth Kozlowski
```
Cris' first name and my last name are misspelled.

kozlowskik replied:
Corrected

griffine commented on src/webapp/README.md:
```diff
@@ -7,21 +7,21 @@ Data Science & Systems Lab (DASSL), Western Connecticut State University (WCSU)
+PROVIDED AS IS. NO WARRANTIES EXPRESSED OR IMPLIED. USE AT YOUR OWN RISK.  
 
+Modified by **Team GEEKS** (*CS298-01 Spring 2019 @ WCSU*)  
+Bruno DaSilva, Elly Griffen, Chris Fitzgerald and Kenneth Kozlowski
```
Cris' first name and my last name are misspelled.

kozlowskik replied:
Corrected

kozlowskik committed:
Fixed Spelling Errors in READMEs

griffine commented:
Other than the spelling errors that were fixed, I think it looks good. I do think that we should be consistent with which names we use (Ken vs Kenneth and Elly vs Eliot) to avoid any possible confusion to others. I have no preference for which name is used for me.

kozlowskik committed:
Fixed Spelling of Names to add Consistancy

Fixed all of our names to make them more consistant in all of the documents in the Repo.
